WooCommerce/WooCommerceTests/ViewRelated/Products/SKU Scanner/ProductSKUBarcodeScannerCoordinatorTests.swift

```diff
@@ -0,0 +1,103 @@
+import TestKit
+import XCTest
+@testable import WooCommerce
+
+final class ProductSKUBarcodeScannerCoordinatorTests: XCTestCase {
+    private var navigationController: UINavigationController!
+    private var window: UIWindow?
+
+    override func setUp() {
+        super.setUp()
+        navigationController = UINavigationController()
+
+        let window = UIWindow(frame: UIScreen.main.bounds)
+        window.rootViewController = UIViewController()
+        window.makeKeyAndVisible()
+        window.rootViewController = navigationController
```

Comment:
Just curious: why do we need to set `rootViewController` twice?

Reply:
Great catch! It should be only set once, updated in: https://github.com/woocommerce/woocommerce-ios/pull/5679/commits/8b82ace0da6fc5bc74053f5f12cb2a0a1373e92a